datafusion/src/catalog/schema.rs

```diff
@@ -57,6 +58,12 @@ pub trait SchemaProvider: Sync + Send {
             "schema provider does not support deregistering tables".to_owned(),
         ))
     }
+
+    /// If supported by the implementation, checks the table exist in the schema provider or not.
+    /// If no matched table in the schema provider, return false.
+    /// Otherwise, return true.
+    #[allow(unused_variables)]
```

Comment:
I think we can remove `#[allow(unused_variables)]` now?

Reply:
yep!!!